### DISCUSSION TITLE
decorator for versioning_classes

### DISCUSSION THREAD

**Discussion by amey589** [encode/django-rest-framework#7838] [Ideas & Suggestions]
I am curious if DRF can add decorator for versioning_classes. I am using 3.12.2 and I do not see its versioning_classes decorator available. Happy to contribute. I am not sure if its intentional to omit versioning_classes decorator though.

**Comment by lovelydinosaur** [comment]
Probably not. Pretty sure we've had this come up in the past and the previous resolution was "we don't particularly want to provide that", although I don't find anything from a "versioning_classes" search.
